### PR TITLE
feat(HDrezka): add presence

### PR DESCRIPTION
**Pull request by env3lope**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

firefox_n54YCpfNjU
Discord_jwTKGJ1F9c

</details>

### CODE REVIEW

File: websites/H/HDrezka/presence.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+const presence = new Presence({
+	clientId: "1191450515670843533",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Где-то на сайте",
+			largeImageKey:
+				"https://cdn.discordapp.com/app-assets/1191450515670843533/1247862870276833292.png",
```
Comment (theusaf) [inline websites/H/HDrezka/presence.ts:9]: Replace these with `https://i.imgur.com/0gTpbcv.png` which is a 512x512 image

File: websites/H/HDrezka/presence.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+			presenceData.buttons = [
+				{
+					label: "Открыть страницу",
+					url: document
+						.querySelector("meta[property='og:url']")
+						.getAttribute("content"),
+				},
+			];
+		} else {
+			presenceData.details = `В поиске ${currentType}a`;
```
Comment (theusaf) [inline websites/H/HDrezka/presence.ts:51]: Is this a typo?
```suggestion
			presenceData.details = `В поиске ${currentType}`;
```

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/H/HDrezka/presence.ts (hunk 2 above)
Comment (env3lope) [reply websites/H/HDrezka/presence.ts]: it's more like bad solution, I've redesigned it

File: websites/H/HDrezka/presence.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	) {
+		if (document.location.pathname.match(/\/(films|series|cartoons)\/.+/)) {
+			presenceData.details = `Смотрит ${currentType}`;
+			presenceData.state = `${
+				document.querySelector(".b-post__title h1").textContent
+			}`;
+			presenceData.largeImageKey =
+				document.querySelector<HTMLImageElement>(".b-sidecover a img").src;
+			presenceData.smallImageKey =
+				"https://cdn.discordapp.com/app-assets/1191450515670843533/1247862870276833292.png";
```
Comment (theusaf) [inline websites/H/HDrezka/presence.ts:40]:
```suggestion
				"https://i.imgur.com/0gTpbcv.png";
```

File: websites/H/HDrezka/presence.ts (hunk 1 above)
Comment (env3lope) [reply websites/H/HDrezka/presence.ts]: done

File: websites/H/HDrezka/presence.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+					label: "Открыть страницу",
+					url: document
+						.querySelector("meta[property='og:url']")
+						.getAttribute("content"),
+				},
+			];
+		} else {
+			presenceData.details = `Ищет ${currentType}`;
+			presenceData.smallImageKey =
+				"https://cdn.discordapp.com/app-assets/1191450515670843533/1191452075100479498.png";
```
Comment (theusaf) [inline websites/H/HDrezka/presence.ts:52]:
```suggestion
				"https://cdn.discordapp.com/app-assets/1191450515670843533/1191452075100479498.png?size=512";
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**